core/templates/pages/exploration-player-page/learner-experience/end-chapter-confetti.component.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the end chapter celebration confetti component.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+
+@Component({
+  selector: 'oppia-end-chapter-confetti',
+  template: require('./end-chapter-confetti.component.html'),
+})
+export class EndChapterConfettiComponent implements OnInit {
+  confettiIsShown: boolean = false;
+  audioUrl: string;
+  celebratoryTadaa = new Audio();
```

Comment:
The variable name here does not seem to convey that it is refering to an audio file. Also, in the future may change the audio file and it might not have a "tadaa" effect. Can we rename it?

Reply:
Done.